examples/2-tier-iis-sql-vm/modules/sql-vm/variables.tf

```diff
@@ -0,0 +1,39 @@
+variable "resource_group_name" {
+  description = "The name of the Resource Group where the Windows Client resources will be created"
+}
+
+variable "location" {
+  description = "The Azure Region in which the Resource Group exists"
+}
+
+variable "prefix" {
+  description = "The Prefix used for the Windows Client resources"
+}
+
+variable "subnet_id" {
+  description = "The Subnet ID which the Windows Client's NIC should be created in"
+}
+
+variable "active_directory_domain" {
+  description = "The name of the Active Directory domain, for example `consoto.local`"
+}
+
+variable "active_directory_username" {
+  description = "The username of an account with permissions to bind machines to the Active Directory Domain"
+}
+
+variable "active_directory_password" {
+  description = "The password of the account with permissions to bind machines to the Active Directory Domain"
+}
+
+variable "admin_username" {
+  description = "The username associated with the local administrator account on the virtual machine"
+}
+
+variable "admin_password" {
+  description = "The password associated with the local administrator account on the virtual machine"
+}
+
+variable sqlvmcount {}
```

Comment:
this syntax (using a variable without the quotes) will break in Terraform 0.12 - it'd be worth adding quotes around this to ensure it's compatible in the future